org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/json/JSonLanguageServer.java

```diff
@@ -108,6 +115,21 @@ private void fillSchemaAssociationsForTypeScript(Map<String, List<String>> assoc
 	private void fillSchemaAssociationsForOmnisharp(Map<String, List<String>> associations) {
 		associations.put("/omnisharp.json", Arrays.asList("http://json.schemastore.org/omnisharp"));
 	}
+	
+	private void fillSchemaAssociationsFromExtensionPoint(Map<String, List<String>> associations) {
+		IConfigurationElement[] conf = Platform.getExtensionRegistry().getConfigurationElementsFor(SCHEMA_EXT);
+		for (IConfigurationElement el : conf) {
+			associations.put(el.getAttribute(NAME_ATTR), collectSchemaURL(el.getChildren(SCHEMA_ELEMENT)));
```

Comment:
why not schema as attribute?
Also, what it 2 distinct plugin want to add different schema for the same name/pattern? In this case, I would expect that the list would be joined.

Reply:
Currently it will be replaced. As I saw in hardcoded part, it's always registered as list so I prepared extension point in same way.

I can reimplement it as append/merge. I'm also thinking about rename `name` to `pattern`.

Reply:
Renaming the name to pattern seems good.
Allowing to merge extensions for same pattern seems good too.
As for children elements vs attributes, I don't have a strong preference. I'll let you defend the structure you prefer.

Reply:
I updated documentation and simplify extension schema